execution_engine/src/core/resolvers/v1_function_index.rs

```diff
@@ -58,6 +58,7 @@ pub(crate) enum FunctionIndex {
     LoadCallStack,
     LoadAuthorizationKeys,
     RandomBytes,
+    DictionaryReadFuncIndex,
```

Comment:
@piotr-dziubecki , please make a 2.0 tech debt ticket to normalize the variant names of this enum